### PR TITLE
List Structure

### PR DESCRIPTION
**Pull request by disc3110**
- Set up project with Webpack
- Create to-do list placeholder
- Create array with to-do tasks objects
- To-do tasks objects with properties:
    - description [string]
    - completed [bool]
    - index [number]
- Populate HTML with elements for each task
- Render tasks on page load

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+.tasks-item {
+  align-items: center;
+  padding: 1rem;
+  border-bottom: 1px solid #eee;
+}
+
+.checkbox {
+  margin-right: 5px;
+}
```
Comment (teekaytech) [inline src/style.css]: - [ ] For this project, there's a specific styling template stated to follow. Kindly use your created CSS to style the list using the [captured video](https://www.youtube.com/watch?v=AcUd-_Yjjqg).  Please note that the video has more features than the current project. Therefore, you don't need to style all the features not implemented yet in the video.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+## Pre requisites
+have npx installed
+
+## Getting Started
+Get a copy of the repository running this command in your OS terminal: `git clone https://github.com/disc3110/to-do-list.git` to get a copy of the project.
+
+
+go to the project folder
+
+run `npm run start`
```
Comment (teekaytech) [inline README.md:26]: **OPTIONAL**

- [ ] Kindly update your Readme file
- I strongly suggest you add the `Getting Started` section, where you state the instructions and code samples that will be needed to clone your project, install its dependencies, and starting the project on a dev environment.
- The live demo link is broken: if you don't have a live demo yet, you can remove this link. It is not a good practice to have broken link(s) in your Readme.
- You may also add a screenshot of the project to your Readme.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @disc3110 :wave:

_Your project is almost ready to go. You've shown a high degree of understanding and professionalism. Your PR title, summary, and commit messages are great. Also, all your linter checks are running with no errors :100:_

__However, you need to make some required changes before moving on to the next assignment.__
> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it :blush: 

### **Required Changes** :recycle: 
Check the comments under the review.

### **Optional Changes** :zap: 
_Every comment with the **OPTIONAL** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


* * *
**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@teekaytech) so I can be notified about your comments and respond._

*REMINDER: As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).*

_**Please make the required changes and submit another review request when you are done.**_

* * *

_Reviewed by:_ [**Taofeek Olalere**](https://github.com/teekaytech)
__Cheers & Happy coding :smiley:__

File: README.md (hunk 2 above)
Comment (Dipeshtwis) [inline README.md:26]: [OPTIONAL[

I strongly recommend to fix this
- After going to the project folder and running `npm run start` would give an error. You have missed one command to include which is `npm install`. Please mention it in your README file.

**Review by Dipeshtwis — APPROVED** [review]
# Hi @disc3110, 

### Status: Approved ✔️

:+1:  Professional repo, with good PR title and description, keep up the good work :muscle:
:+1: You have implemented the suggested changes of the previous reviewer.

Your project is complete!  There are some optional changes I've suggested to you to make your README professional with right instructions, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- You can use screenshot in README file so that other user would know how your projects look like in one scroll.


**_Also check the comments under the review_**


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Dipesh Kumar_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by disc3110** [comment]
I made the items draggable so the user can change the order of the thing as he/she wants. it updates the array values its time you change the order or mark the checkbox

- Add 2 new JavaScript files and import them as modules:
  - one will contain methods related to drag / drop and sorting functionality.
  - the other will contain methods related to the status updates (completed: true / false).
- Add event listeners to list items (dragstart, dragover, drop) and to checkbox (change).
- Update items object's values for index and completed keys upon user actions.
- Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.

**Comment by mricanho** [comment]
Hi @disc3110, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project in the same Pull Request as the last one, remember that every new project must have it's own PR.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**